Adding `pyleniumio` from PyPI

sugatoray opened the pull request:
<!--
Thank you very much for putting in this recipe PR!

This repository is very active, so if you need help with
a PR or once it's ready for review, please let the right people know.
There are language-specific teams for reviewing recipes.

Currently available teams are:
- python `@conda-forge/help-python`
- python/c hybrid `@conda-forge/help-python-c`
- r `@conda-forge/help-r`
- java `@conda-forge/help-java`
- nodejs `@conda-forge/help-nodejs`
- c/c++ `@conda-forge/help-c-cpp`
- perl `@conda-forge/help-perl`
- Julia `@conda-forge/help-julia`
- ruby `@conda-forge/help-ruby`

If your PR doesn't fall into those categories please contact
the full review team `@conda-forge/staged-recipes`.

Due to GitHub limitations first time contributors to conda-forge are unable
to ping these teams. You can [ping the team][1] using a special command in
a comment on the PR to get the attention of the `staged-recipes` team. You can
also consider asking on our [Gitter channel][2] if your recipe isn't reviewed promptly.

[1]: https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team
[2]: https://gitter.im/conda-forge/conda-forge.github.io
-->

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged. `NA`
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/pyleniumio```) and found it was in an excellent condition.

sugatoray commented:
## Notes on Current Error

## https://github.com/conda-forge/staged-recipes/pull/17240/commits/6591b6783e36b21a76ca6f297ce8d363f476b869

Current :no_entry: error [(error-stack)](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=426789&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1379), despite `pyleniumio` being downloaded and extracted successfully.

```sh
poetry.core.masonry.utils.module.ModuleOrPackageNotFound: No file/folder found for package pyleniumio
```

<details>
<summary>Expand for more details on the error</summary>
<p>

```sh
Success
Extracting download
source tree in: /home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/work
export PREFIX=/home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_
export BUILD_PREFIX=/home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/_build_env
export SRC_DIR=/home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/work
Using pip 21.3.1 from $PREFIX/lib/python3.10/site-packages/pip (python 3.10)
Non-user install because user site-packages disabled
Ignoring indexes: https://pypi.org/simple
Created temporary directory: /tmp/pip-ephem-wheel-cache-w5u3g9zu
Created temporary directory: /tmp/pip-req-tracker-s4dac9ua
Initialized build tracking at /tmp/pip-req-tracker-s4dac9ua
Created build tracker: /tmp/pip-req-tracker-s4dac9ua
Entered build tracker: /tmp/pip-req-tracker-s4dac9ua
Created temporary directory: /tmp/pip-install-kdvpu5xt
Processing $SRC_DIR
  Added file://$SRC_DIR to build tracker '/tmp/pip-req-tracker-s4dac9ua'
  Created temporary directory: /tmp/pip-modern-metadata-hjqir85y
  Preparing metadata (pyproject.toml): started
  Running command /home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/bin/python /home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py prepare_metadata_for_build_wheel /tmp/tmpw3nsn63l
  Traceback (most recent call last):
    File "/home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
      main()
    File "/home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 164, in prepare_metadata_for_build_wheel
      return hook(metadata_directory, config_settings)
    File "/home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/poetry/core/masonry/api.py", line 44, in prepare_metadata_for_build_wheel
      builder = WheelBuilder(poetry)
    File "/home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/poetry/core/masonry/builders/wheel.py", line 57, in __init__
      super(WheelBuilder, self).__init__(poetry, executable=executable)
    File "/home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/poetry/core/masonry/builders/builder.py", line 86, in __init__
      self._module = Module(
    File "/home/conda/staged-recipes/build_artifacts/pyleniumio_1639218273024/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/poetry/core/masonry/utils/module.py", line 63, in __init__
      raise ModuleOrPackageNotFound(
  poetry.core.masonry.utils.module.ModuleOrPackageNotFound: No file/folder found for package pyleniumio
  Preparing metadata (pyproject.toml): finished with status 'error'
```

</p>
</details>

![image](https://user-images.githubusercontent.com/10201242/145746905-3a0fc134-806e-414e-a9b8-489a2245f828.png)

---

sugatoray commented:
keep this open
